### PR TITLE
ACA-1600: Adding admin password requirements

### PR DESCRIPTION
**Pull request by Yaish25491**
Adding password requirements to README file

### CODE REVIEW

**Comment by Yaish25491** [comment]
@nirarg @p3ck @prabinovRedhat  Please review

File: roles/azure_manage_postgresql/README.md [hunk 1]
```diff
@@ -29,7 +29,13 @@ Role Variables
 * **azure_manage_postgresql_postgresql_enforce_ssl**: Enable SSL enforcement. Default: False
 * **azure_manage_postgresql_postgresql_storage_autogrow**: Enable storage autogrow. Default: False
 * **azure_manage_postgresql_postgresql_admin_username**: The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation).
-* **azure_manage_postgresql_postgresql_admin_password**: The password of the administrator login. When this is not defined, the role will generated a password that can be read later in the variable name.
+* **azure_manage_postgresql_postgresql_admin_password**: The password of the administrator login. When this is not defined, the role will generated a password that can be read later in the variable name.  
+**The supplied password must be between 6-72 characters long and must satisfy at least 3 of password complexity requirements from the following**:
```
Comment (nirarg) [inline roles/azure_manage_postgresql/README.md:33]: I don't think this should be bold, but not critical

**Review by nirarg — APPROVED** [review]

File: changelogs/fragments/20240707-postgresql_admin_manage_password.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - Adding admin password requirements of azure_manage_postgresql_postgresql_admin_password role to README file
```
Comment (prabinovRedhat) [inline changelogs/fragments/20240707-postgresql_admin_manage_password.yml:2]:
```suggestion
  - Adding admin password requirements of azure_manage_postgresql role to README file
```

File: roles/azure_manage_postgresql/README.md [hunk 3]
```diff
@@ -29,7 +29,13 @@ Role Variables
 * **azure_manage_postgresql_postgresql_enforce_ssl**: Enable SSL enforcement. Default: False
 * **azure_manage_postgresql_postgresql_storage_autogrow**: Enable storage autogrow. Default: False
 * **azure_manage_postgresql_postgresql_admin_username**: The administrator's login name of a server. Can only be specified when the server is being created (and is required for creation).
-* **azure_manage_postgresql_postgresql_admin_password**: The password of the administrator login. When this is not defined, the role will generated a password that can be read later in the variable name.
+* **azure_manage_postgresql_postgresql_admin_password**: The password of the administrator login. When this is not defined, the role will generated a password that can be read later in the variable name.
```
Comment (prabinovRedhat) [inline roles/azure_manage_postgresql/README.md:32]: When this is not defined, the role will generated a password - I'm not sure that is correct

File: changelogs/fragments/20240707-postgresql_admin_manage_password.yml (hunk 2 above)
Comment (prabinovRedhat) [reply changelogs/fragments/20240707-postgresql_admin_manage_password.yml:2]: or
```suggestion
  - Adding admin password requirements of azure_manage_postgresql_postgresql_admin_password var to README file of azure_manage_postgresql role
```

Comment (Yaish25491) [reply changelogs/fragments/20240707-postgresql_admin_manage_password.yml]: applied this change:  Adding admin password requirements of azure_manage_postgresql role to README file

File: roles/azure_manage_postgresql/README.md (hunk 1 above)
Comment (Yaish25491) [reply roles/azure_manage_postgresql/README.md]: Bold removed

**Comment by prabinovRedhat** [comment]
I see some of the checks failed

**Comment by nirarg** [comment]
Great work, thanks
Its ready for merge

**Pull request merged**